src/community/wfs3/src/main/java/org/geoserver/wfs3/DefaultWebFeatureService30.java

```diff
@@ -191,4 +192,9 @@ public FeatureCollectionResponse getTile(org.geoserver.wfs3.GetFeatureType reque
 
         return response;
     }
+
+    @Override
+    public TilingSchemeDetailDocument tilingSchemeDetail(TilingSchemeDetailRequest request) {
```

Comment:
TilingSchemeDescriptionDocument/describeTilingScheme (matching the terminology already present in WFS, .e.g., DescribeFeatureType)